torch_sim/math.py

```
@@ -1016,7 +1019,7 @@ def batched_vdot(
     if batch_indices.min() < 0:
         raise ValueError("batch_indices must be non-negative")
 
-    output = torch.zeros(batch_indices.max() + 1, dtype=x.dtype, device=x.device)
+    output = torch.zeros(batch_indices.max() + 1, dtype=x.dtype, device=x.device) # type: ignore
```

Ruff (0.12.2)

Use specific rule codes when ignoring type issues

(PGH003)